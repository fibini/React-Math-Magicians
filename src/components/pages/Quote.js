import React from 'react';

const Quote = () => (
  <div
    style={{
      backgroundColor: '#e0e0e0',
      border: 2,
      borderStyle: 'solid',
      borderColor: 'lightgrey',
      borderRadius: 10,
    }}
  >
    <p style={{ display: 'flex', justifyContent: 'center' }}>
      Mathematics is not about numbers, equations, computations or algorithms:
      it is about understanding. -William Paul Thurston
    </p>
  </div>
);
export default Quote;
