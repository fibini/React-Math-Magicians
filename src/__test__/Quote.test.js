import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Quote />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
