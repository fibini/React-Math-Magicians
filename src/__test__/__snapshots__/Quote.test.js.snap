// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
